src/TypeDefinition.js

```diff
@@ -381,6 +381,7 @@ export type NavigationTransitionProps = {
   // is the index of the scene
   scene: NavigationScene,
   index: number,
+  navigation: NavigationScreenProp<NavigationRoute, NavigationAction>,
```

Comment:
@ericvicenti you are right. In order to make flow pass, we would have to define `navigation` directly inside `Transitioner` `buildTransitionProps`. Do you think it makes sense? Not sure how that corresponds to the `_getChildNavigation()` that seems to have some memoization. Anyway, the only place where we don't have `navigation` is all calls prior to _render(). Any ideas?